### PR TITLE
Secure

### PR DESCRIPTION
**Pull request by ericshermancs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,5 +25,8 @@
   "bugs": {
     "url": "https://github.com/Grade-Notifier/GN-Core/issues"
   },
-  "homepage": "https://github.com/Grade-Notifier/GN-Core#readme"
+  "homepage": "https://github.com/Grade-Notifier/GN-Core#readme",
```
Comment (Huddie) [inline package.json]: Can we get rid of this?

File: src/core/grade_notifier.py [hunk 2]
```diff
@@ -251,10 +251,7 @@ def start_notifier():
 
             cursor.execute(f'UPDATE Users SET lastUpdated = NOW() WHERE id={__id};')
             
-            fern = Fernet(os.getenv('DB_ENCRYPTION_KEY').encode('utf-8'))
-            decrypted_password = fern.decrypt(
-                encrypted_password.encode('utf-8')
-            ).decode()
+            decrypted_password = decrypt(encrypted_password, 'test_keys/private.pem')
```
Comment (Huddie) [inline src/core/grade_notifier.py]: We dont want to pass in test_keys, that's just for testing, we want env variables

File: src/core/initializegn.py [hunk 3]
```diff
@@ -128,9 +129,10 @@ def main():
             )
             return
 
-        f = Fernet(key)
-        password = f.decrypt(encrypted_password.encode('utf-8')).decode()
+        password = decrypt(encrypted_password, 'test_keys/private.pem')
```
Comment (Huddie) [inline src/core/initializegn.py]: Dont wanna pass in test_keys, we want env variables

File: src/core/initializegn.py [hunk 4]
```diff
@@ -128,9 +129,10 @@ def main():
...
+        # password = f.decrypt(encrypted_password.encode('utf-8')).decode()
```
Comment (Huddie) [inline src/core/initializegn.py]: dont comment out, delete it. It's garbage now right?

File: src/helper/security.py [hunk 5]
```diff
@@ -1,33 +1,34 @@
 from Crypto.PublicKey import RSA
 from Crypto.Cipher import PKCS1_OAEP
+from Crypto.Signature import PKCS1_PSS
+from Crypto.Hash import SHA
```
[the remark this answers is not in the document]
Comment (Huddie) [reply src/helper/security.py]: Lets remove this if its unneeded (this was my bad)

File: package.json (hunk 1 above)
Comment (ericshermancs) [reply package.json]: I guess? I didnt make this

Comment (Huddie) [reply package.json]: jsrsasign this is not you? @ericshermancs

**Pull request merged**